vyper/parser/context.py

```diff
@@ -48,8 +47,6 @@ def __init__(
         self.in_range_expr = False
         # Is the function payable?
         self.is_payable = is_payable
-        # Original code (for error pretty-printing purposes)
-        self.origcode = origcode
```

Comment:
Basically, this is no longer useful because all errors are caught upstream in context?

Reply:
Correct.